spec/integ/matrix-client-crypto.spec.js

```diff
@@ -24,9 +24,6 @@ limitations under the License.
  * See also `megolm.spec.js`.
  */
 
-"use strict";
-import 'source-map-support/register';
```

Comment:
We don't need these imports anymore - they actually make the test's sourcemaps wrong.